src/org/traccar/database/SimpleObjectManager.java

```diff
@@ -65,11 +67,7 @@ protected final void clearItems() {
         items.clear();
     }
 
-    protected final void putItem(long itemId, BaseModel item) {
-        items.put(itemId, item);
-    }
-
-    protected final void removeCachedItem(long itemId) {
+    protected void removeCachedItem(long itemId) {
```

Comment:
I think it makes sense to keep `updateCachedItem` and `removeCachedItem` next to each other.